ground/src/main/java/com/google/android/ground/ui/map/gms/GoogleMapsFragment.kt

```diff
@@ -209,15 +211,16 @@ class GoogleMapsFragment : SupportMapFragment(), MapFragment {
     }
   }
 
-  private fun onMarkerClick(marker: Marker): Boolean =
-    if (getMap().uiSettings.isZoomGesturesEnabled) {
-      markers[marker]?.let { markerClicks.onNext(it) }
+  private fun onClusterItemClick(item: LocationOfInterestClusterItem): Boolean {
+    return if (getMap().uiSettings.isZoomGesturesEnabled) {
+      markerClicks.onNext(MapLocationOfInterest(item.locationOfInterest))
```

Comment:
Not sure I understand the relation between map pins and cluster items here.

Reply:
This function is a no-op change. Basically, this is the same function we previously had, we just need to pass a different type since the cluster manager now handles marker interactions (It's a bit weird, but it's how the clustering API works).

Reply:
Got it!